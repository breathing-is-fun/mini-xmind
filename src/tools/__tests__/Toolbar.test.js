import React from 'react';
import { mount } from 'enzyme';
const testDist = process.env.LIB_DIR === 'dist';
const Toolbar = testDist
  ? require('../../../dist/lib/toolbar').default
  : require('../Toolbar').default;
import 'nino-cli/scripts/setup';

describe('Toolbar', () => {
  it('toolbar render correctly', () => {
    expect(mount(<Toolbar />)).toMatchSnapshot();
  });
});
