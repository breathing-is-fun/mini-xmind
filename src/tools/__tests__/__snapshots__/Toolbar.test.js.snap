// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar toolbar render correctly 1`] = `
<Toolbar>
  <div
    className="Toolbar"
  >
    <ul>
      <li
        draggable={true}
        key="tool-0"
        onDragStart={[Function]}
      >
        <Icon
          style={
            Object {
              "fontSize": 20,
            }
          }
          type="minus"
        >
          <i
            className="anticon anticon-minus"
            style={
              Object {
                "fontSize": 20,
              }
            }
          >
            <IconReact
              className=""
              type="minus-o"
            >
              <svg
                aria-hidden="true"
                className=""
                data-icon="minus"
                fill="currentColor"
                height="1em"
                key="svg-minus"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M872 474H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h720c4.4 0 8-3.6 8-8v-60c0-4.4-3.6-8-8-8z"
                  key="svg-minus-svg-0"
                />
              </svg>
            </IconReact>
          </i>
        </Icon>
      </li>
      <li
        draggable={true}
        key="tool-1"
        onDragStart={[Function]}
      >
        <Icon
          style={
            Object {
              "fontSize": 20,
            }
          }
          type="border"
        >
          <i
            className="anticon anticon-border"
            style={
              Object {
                "fontSize": 20,
              }
            }
          >
            <IconReact
              className=""
              type="border-o"
            >
              <svg
                aria-hidden="true"
                className=""
                data-icon="border"
                fill="currentColor"
                height="1em"
                key="svg-border"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-40 728H184V184h656v656z"
                  key="svg-border-svg-0"
                />
              </svg>
            </IconReact>
          </i>
        </Icon>
      </li>
      <li
        draggable={true}
        key="tool-2"
        onDragStart={[Function]}
      >
        <Icon
          style={
            Object {
              "fontSize": 20,
            }
          }
          type="tag"
        >
          <i
            className="anticon anticon-tag"
            style={
              Object {
                "fontSize": 20,
              }
            }
          >
            <IconReact
              className=""
              type="tag-o"
            >
              <svg
                aria-hidden="true"
                className=""
                data-icon="tag"
                fill="currentColor"
                height="1em"
                key="svg-tag"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M938 458.8l-29.6-312.6c-1.5-16.2-14.4-29-30.6-30.6L565.2 86h-.4c-3.2 0-5.7 1-7.6 2.9L88.9 557.2a9.96 9.96 0 0 0 0 14.1l363.8 363.8c1.9 1.9 4.4 2.9 7.1 2.9s5.2-1 7.1-2.9l468.3-468.3c2-2.1 3-5 2.8-8zM459.7 834.7L189.3 564.3 589 164.6 836 188l23.4 247-399.7 399.7zM680 256c-48.5 0-88 39.5-88 88s39.5 88 88 88 88-39.5 88-88-39.5-88-88-88zm0 120c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32z"
                  key="svg-tag-svg-0"
                />
              </svg>
            </IconReact>
          </i>
        </Icon>
      </li>
    </ul>
  </div>
</Toolbar>
`;
